config/nrfconnect/nrfconnect-app.cmake

```diff
@@ -53,6 +53,9 @@ endif()
 # ==================================================
 find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 
+# TODO: temporary fix forcing time_t size to be equal long size - remove it after merging fix from Zephyr project.
```

Comment:
Please make sure there's an issue tracking this todo.

Reply:
Ok, good point, done.